docker/intel-dgpu-driver.Dockerfile

```diff
@@ -30,6 +30,8 @@ RUN git clone -b 23WW06.5_555 --single-branch https://github.com/intel-gpu/intel
 
```

Comment:
Since it is a workaround for KMM issue. And our docker file actually is right. 
Can we use something like
ARG KERNEL_FULL_VERSION = ${KERNEL_VERSION}

And after the KMM issue is fixed, we can simply remove the default value of KERNEL_FULL_VERSION

Reply:
We can do the above. It would require us to have the following:
`ARG KERNEL_VERSION`
`ARG KERNEL_FULL_VERSION = ${KERNEL_VERSION}`

Let me test it and if it works we can go with this approach.

Reply:
Done. Tested, the approach is working correctly. Ready to merge.